packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -348,6 +348,7 @@ export class FeatureDevController {
             type: 'answer',
             tabID: tabID,
             canBeVoted: true,
+            snapToTop: true,
```

Comment:
Is this only required for plan generation?

Reply:
yes, plan is the only "long" card we have. With code generation the card is normally "fits" the scrren